### DISCUSSION TITLE
Password reset emails are working, but automated report emails are not

### DISCUSSION THREAD

**Discussion by Shotster** [plausible/analytics#4060] [Self-Hosted Support]
I'm using **v2.1.0-rc.0** in a Docker setup, and everything is working great, except for the fact that automated analytics reports are not being sent.

I'm using the **Bamboo.SendGridAdapter**, and I get password reset emails as expected. However, I'm not getting weekly or monthly analytics reports as configured in the UI - not even after adding `CRON_ENABLED=true` to the environment variables.

```
BASE_URL=https://analytics.my-domain.com
SECRET_KEY_BASE=<my-key-base>
TOTP_VAULT_KEY=<my-vault-key>

MAILER_EMAIL='info@my-domain.com'
MAILER_NAME='Analytics Report'
MAILER_ADAPTER='Bamboo.SendGridAdapter'
SENDGRID_API_KEY='<my-sendgrid-api-key>'
CRON_ENABLED=true
```

Any ideas on how to troubleshoot or what to try next?

**Comment by ruslandoga** [comment]
👋 @Shotster 

Is there anything interesting in `docker compose logs`? Or maybe there are errors in the background jobs?

```console
$ docker compose exec plausible_db psql -h localhost -U postgres
```
```sql
select * from oban_jobs;
```

**Reply by ruslandoga** [reply]
Should be fixed in https://github.com/plausible/analytics/pull/4061

**Reply by ruslandoga** [reply]
The fix is released in v2.1.0-rc.1: https://github.com/plausible/analytics/discussions/4084
